cli/common.mli

```diff
@@ -5,13 +5,18 @@ module Logs : sig
 end
 
 module Arg : sig
+  val named : ('a -> 'b) -> 'a Cmdliner.Term.t -> 'b Cmdliner.Term.t
```

Comment:
Why putting this in `Arg` ? This module contains all the arguments (and not overloading `Cmdliner.Arg`) and this is not an argument itself. What do you think of moving it into `Common` ?

Reply:
I think that's where it belongs, we use it to build arguments consistently across the commands.